### DISCUSSION TITLE
Any plans to make "Experimental third-party storage partitioning" flag always true in future? if yes, when it possibly be?

### DISCUSSION THREAD

**Discussion by teja-0414** [privacysandbox/privacy-sandbox-dev-support#151] [Phasing out third-party cookies] [answered]
I am using Chrome 116 with flag 'Experimental third-party storage partitioning' enabled.
My main site a1.com embeds a site a2.com using iframe to share cookies, storage objects which holds authentication tokens. Site aw.com trying to get the object from a1.com but the object cannot be accessed with which a2.com is breaking.
After searching a lot over the web I came across this flag "Experimental third-party storage partitioning"
Any future plans of google which will make this always true, so as plan a buffer to make my project work even when this flag is enabled?

**Comment by miketaylr** [comment, marked as answer]
Hi @teja-0414. Yes, we are current in the process of setting this feature to always true: https://groups.google.com/a/chromium.org/g/blink-dev/c/24hK6DKJnqY. Right now it's true for 50% of Chrome users.

There is a deprecation trial available if needed: https://developer.chrome.com/blog/storage-partitioning-deprecation-trial/

**Reply by teja-0414** [reply]
Will this flag be present permanently or will partitioning of third party storage done always irrespective of the flag(deprecating or deleting) in future?

**Reply by miketaylr** [reply]
We will not likely keep the flag around after the feature ships to 100% of Chrome Stable users (in the coming weeks). So, if you need more time to migrate to a different solution, the Deprecation Trial in the above link is a better solution.
